doc/newsboat.txt

```diff
@@ -1001,6 +1001,7 @@ Identifier:Meaning
 [[podlist-format-S]]<<podlist-format-S,+S+>>:Status of download, displays one of the folowing; "queued", "downloading", "ready", "canceled", "deleted", "incomplete", "played", "finished" or "failed"
 [[podlist-format-u]]<<podlist-format-u,+u+>>:Url of the download
 [[podlist-format-F]]<<podlist-format-F,+F+>>:Filename of the download
```

Comment:
Can you update this line to highlight the difference between this and %b? Right now, it's not entirely clear how "filename" is different from the "basename".

Reply:
That's a good idea. Thanks.